build/localnet/cluster-manager/utils.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	v1 "k8s.io/api/core/v1"
+)
+
+func isValidator(service *v1.Service) bool {
+	return service.Labels["v1-purpose"] == "validator"
+}
+
+func extractValidatorId(validatorName string) string {
+	if len(validatorName) >= 3 {
+		return validatorName[len(validatorName)-3:]
+	}
+	return validatorName
+}
+
+func getPrivateKey(validatorKeysMap map[string]string, validatorId string) cryptoPocket.PrivateKey {
```

Comment:
https://github.com/pokt-network/pocket/pull/491/files#r1108003092
> TOL: Creating a type for `map[string]string`

Overkill IMHO

Reply:
```suggestion
// TODO: Create a type for `validatorKeyMap` and document what the expected key-value types contain
func getPrivateKey(validatorKeysMap map[string]string, validatorId string) cryptoPocket.PrivateKey {
```

Reply:
Stating thinking about it more and think it might be useful. Outside the scope of this PR though